src/tests/Dictionary/DictionaryModal.test.jsx

```diff
@@ -189,6 +189,52 @@ describe('Test suite for dictionary modal', () => {
     expect(wrapper3.instance().state.errors).toEqual({});
   });
 
+  it('it should handle undefined submit error response', async () => {
```

Comment:
Is this related to the ticket?

Reply:
Not exactly, after implementing the changes, the test coverage for that particular test dropped. I had to write that test to increase the coverage.